docs/General/on-the-server.rst

```diff
@@ -0,0 +1,109 @@
+..
+    SPDX-License-Identifier: GPL-3.0-or-later
+    SPDX-FileCopyrightText: 2022 louis94 <m_louis30@yahoo.com>
+
+Freeciv21 on the Server
+***********************
+
+In order to play Freeciv21 on the network, one machine needs to act as the *server*: the state of
+the game lives on the server and players can connect to play. There are many ways to operate such a
+server; this page gives an introduction to the topic.
+
+In order to run a server, you need a computer that other players can reach over the network. If
+you're planning to play at home, this can usually be any machine connected to your WiFi or local
+cabled network. Otherwise, the easiest is to rent a small server from a hosting provider. We
+strongly recommend that you choose a Linux-based server, as that is what we have experience with.
+You will need the ability to run your own programs on the server, so SSH access is a must. Apart
+from that, Freeciv21 is quite light on resources so you will hardly hit the limits of even the
+cheapest options.
+
+Whether you choose to use your own machine or to rent one, the basic principle of operating a
+server is the same: you need to run a program called ``freeciv21-server`` for as long as the game
+will last. This program will wait for players to connect and handle their moves in the exact same
+way as in a single player game. In fact, Freeciv21 always uses a server, even when there is only
+one player!
```

Comment:
I'd rephrase this as "In fact, Freeciv21 uses an internal server for single player games!", as it's referencing this. "Even when there is only one player" might be confused for a single player being connected to a regular server. Then again, this duplicates the "single player game" term, but I think it's fine here.

Reply:
So this interesting. On one hand with the client manual we are thinking of moving away from the explicit understanding of the "game" being client/server in nature, but then here we go and make it explicit. The FAQ does this as well. We might need to discuss how we want to handle this. I am personally fine with a consistent message that fc21 follows a client/server model. We can explain what that means for users who may not be super-technical and still accomplish our goals with documentation.

Reply:
I think in this case there is no way around mentioning the client/server, but I still don't use the word *client*. I wrote this paragraph as if the reader had never heard of the server before, so they (sight) can understand it here since it's usually not mentioned.

The FAQ could contain an entry *What's the "server"?* or similar. I agree it would be good to discuss the structure of the upcoming server manual before it gets written :D

Reply:
I like the idea of another question in the FAQ and we def want to discuss more about change in name of the current client manual as part of upcoming server manual.